powersimdata/input/abstract_grid.py

```diff
@@ -43,3 +44,19 @@ def storage_template():
         "terminal_max": None,
     }
     return storage
+
+
+class AbstractGridFactory:
```

Comment:
Consider naming this simply `GridFactory`. Other than being more concise, I think the `Abstract` prefix implies the class itself is abstract, vs producing an `AbstractGrid`.

Reply:
+1 on renaming to `GridFactory` rather than `AbstractGridFactory`.

Reply:
I'm not opposed to renaming, but I think `GridFactory` is a bit confusing as well, as in the current implementation `AbstractGridFactory` actually caches objects like `TAMU`, `FromREISE`, and `FromREISEjl`, which are all `AbstractGrid`'s, and doesn't actually cache the `Grid` class itself. What about `GridDataFactory`?